.travis.yml

```diff
@@ -8,6 +8,9 @@ sudo: required
 dist: trusty
```

Comment:
nit: feel free to add your company to the copyright list if you want ;-)

Reply:
@grahamwhaley : Added. Thanks!